### PR TITLE
Add 04-21-2023 proposal.

### PR DESCRIPTION
**Pull request by reem**
IPFS Preview: https://app.aave.com/governance/ipfs-preview/?ipfsHash=0x8e036cb7b5c10028564ba8d16c19e1b8bda48fa41edb904caec571908a76b1d3

In accordance with https://governance.aave.com/t/gauntlet-recommendations-for-eurs-on-v3-polygon/12798

### CODE REVIEW

File: diffs/preAaveV3PolCapsUpdate_20230421Change_postAaveV3PolCapsUpdate_20230421Change.md
Comment (jmomx) [inline diffs/preAaveV3PolCapsUpdate_20230421Change_postAaveV3PolCapsUpdate_20230421Change.md:1]: these links in the comments are super helpful thanks 
image

**Review by jmomx — APPROVED** [review]
short but sweet - looks good

**Review by reichiang — APPROVED** [review]
lgtm

**Comment by reem** [comment]
Propose transaction simulation: https://dashboard.tenderly.co/public/safe/safe-apps/simulator/c16b4f61-8aac-462f-bcb0-bb4114398eea

**Review by rustboyar — APPROVED** [review]

**Pull request merged**